setup.py

```diff
@@ -39,7 +39,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.5",
+        "pytket ~= 1.5.2",
```

Comment:
This will restrict to 1.5.x. I think it should be `~= 1.5, >= 1.5.2`, That way 1.6 will be allowed.

Reply:
(TBH I'm not completely sure, but I think that's how it works.)